app/assets/images/payment_icons/abitab.svg

```diff
@@ -0,0 +1,75 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="38" height="24" role="img" viewBox="0 0 38 24" x="0px" y="0px" style="enable-background:new 0 0 38 24;" xml:space="preserve" aria-labelledby="pi-abitab">
```

Comment:
Please read the [contributions guideline](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

- Remove any unused `id` attributes
- Remove any unused `class` attributes
- Run the svg code through `svgo`

Reply:
Hi @yanikleiman ,

My mistake. There isn't any unused it attribute.

For the `class` attributes, we want all `<styles>` to be converted into inline attribute. What it means is the following code:

```
<style type="text/css">
  .st1{fill:#FFFFFF;}
</style>
<path xmlns="http://www.w3.org/2000/svg" class="st1" d="M35,1c1.1,0,2,0.9,2,2v18c0,1.1-0.9,2-2,2H3c-1.1,0-2-0.9-2-2V3c0-1.1,0.9-2,2-2H35"/>
```

will become

```
<path xmlns="http://www.w3.org/2000/svg" fill="FFFFFF" d="M35,1c1.1,0,2,0.9,2,2v18c0,1.1-0.9,2-2,2H3c-1.1,0-2-0.9-2-2V3c0-1.1,0.9-2,2-2H35"/>
```